### ISSUE TITLE
Projects Do Not Build

### ISSUE THREAD

**Issue by adstro** [gast-lib/gast-lib#2]
As reported by Noah Feehan, the current projects do not build after a clone. It looks like a problem with the generated R class.

**Comment by adstro** [comment]
Are people still having this problem?

**Comment by AKAMEDIASYSTEM** [comment]
Yes, I just pulled this repo today and there are several problems that prevent running the app.

The one I'm working on right now is that values-v11/styles.xml refers to an unknown resource ("@android:style/Theme.Holo")

Additionally, it seems there are conflicts stemming from a mismatch in the minimum API level supported by the project (there's a HONEYCOMB reference somewhere, yet min. API level is pre-Honeycomb)
